ios/app/watchos/extension/JitsiMeetContext.swift

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright @ 2018-present 8x8, Inc.
+ * Copyright @ 2017-2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+class JitsiMeetContext {
+    private var dictionary : [String : Any]
+
+    var joinConferenceURL : String? = nil;
+
+    init() {
+        dictionary = [:]
+    }
+  
+    init(context: [String : Any]) {
+        dictionary = context
+    }
+  
+    init(jmContext: JitsiMeetContext) {
+        dictionary = jmContext.dictionary
+        joinConferenceURL = jmContext.joinConferenceURL
+    }
+  
+    var conferenceURL : String {
+        get {
+            return String(dictionary["conferenceURL"] as? NSString ?? "NULL")
```

Comment:
We should make this return `String?` instead of using `"NULL"` as the placeholder

Reply:
The thing is that I don't know how to send `nil` as the conferenceURL from JavaScript and then we will have to handle both `NULL` string and `nil` in all places instead of just the string `NULL` for the no conference URL case.

Reply:
What happens if we send the conferenceURL as nil on the Swift side? Is the key missing? (that would be another way to do it). The "NULL" string was always a hack, let's try to solve it.

Reply:
Actually `undefined` translates to `nil` so I will try to get rid of the `NULL` string